var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -125,6 +133,18 @@ def install(self, spec, prefix):
         sh = which('sh')
         sh('./autogen.sh')
 
+        if self.get_arch() == 'ppc64le':
+            # update the config.sub/guess files
+            with working_dir("config"):
+                # get new config.guess and config.sub files
+                print 'Backing up existing config.[sub|guess] files\n'
+                subprocess.call("mv config.sub config.sub.orig", shell=True)
+                subprocess.call("mv config.guess config.guess.orig",
+                                shell=True)
+                print 'Downloading lastest config.[sub|guess] files\n'
+                subprocess.call("wget -O config.sub 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'", shell=True)
+                subprocess.call("wget -O config.guess 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'", shell=True)
```

Comment:
Packages shouldn't fetch things manually like this, as it isn't checksummed or trusted (this is raw `http`), and we can't re-route it to fetch from a mirror on an airgapped network.

If you must fetch something remotely, use either the [`patch()`](https://spack.readthedocs.io/en/latest/packaging_guide.html#patch) directive, which will fetch a patch and check it against a checksum, OR for other files you can use the [resource(https://spack.readthedocs.io/en/latest/packaging_guide.html#resources-expanding-extra-tarballs)] directive.

But in this case you don't have to do either.  If you change the Adios package to extend [`AutotoolsPackage`](https://spack.readthedocs.io/en/latest/packaging_guide.html#implementing-the-installation-procedure) instead of `Package`, our `AutotoolsPackage` subclass already copies in the latest `config.guess` from the system.  We've used this to build a number of packages on our `ppc64le` machines.